docs/decisions/0011-fhirserver.md

```diff
@@ -0,0 +1,145 @@
+# 11. FHIR Standards and FHIR Server
+
+Date: 2022-06-28
+
+## Status
+
+Accepted
+
+## Context and Problem Statement
+
+In healthcare, data is gathered from many different sources. Hospitals, prisons, schools, pharmacies, and more all gather healthcare data that state and local territories (STLTs) need to gather to report public health data accurately. In order to ingest data from these different reporting sites, there needed to be a way to standardize the data across all departments of health. 
+
+Previously, HL7 standards were the solution to this issue, but the problems of HL7 was that it was hard to use with modern internet architecture as well as being humanly unreadable. FHIR was created to solve both of these issues. 
```

Comment:
`the problems of HL7 was that it was hard`
->
`the problem with HL7 is that it is hard`

Reply:
changed